iina/MainMenuActions.swift

```diff
@@ -45,6 +45,12 @@ class MainMenuActionHandler: NSResponder {
     }
   }
 
+  @objc func menuShowCurrentFileInFinder(_ sender: NSMenuItem) {
+    guard let url = player.info.currentURL else { return }
+    guard !player.info.isNetworkResource else { return }
```

Comment:
Could combine these.
```suggestion
    guard let url = player.info.currentURL, !player.info.isNetworkResource else { return }
```

Reply:
I prefer the explicit form, but I don't know the Swift conventions. I followed your suggestion.